modules/microshift-nodeport-unreachable-workaround.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * microshift_troubleshooting/microshift-known-issues.adoc
+:_content-type: PROCEDURE
+[id="microshift-nodeport-unreachable-workaround_{context}"]
+= Manually restarting the `ovnkube-master` pod to resume node port traffic
+
+There is an issue with new installs of {product-title} in which the `NodePort` service traffic might be stopped. To troubleshoot this issue, manually restart the `ovnkube-master` pod in the `openshift-ovn-kubernetes` namespace.
+
+.Prerequisites
+
+* The OpenShift CLI (`oc`) is installed.
+* A cluster installed on infrastructure configured with the OVN-Kubernetes network plugin.
+* Access to the `kubeconfig` file.
+* The KUBECONFIG environment variable is set.
+
+.Procedure
+
+Run the commands listed in each step that follows to restore the `NodePort` service traffic after a new {product-title} install:
+
+. Find the name of the ovn-master pod that needs to be restarted by running the following command:
```

Comment:
You could change "needs to be restarted" to active voice with "that you want to restart"